packages/client/src/components/reports/scape/ScapeReportViewer.tsx

```diff
@@ -85,6 +86,22 @@ export default class ScapeReportViewer extends ConnectedComponent<Props, Stores,
         this.stores.configEditorStore.setCanaryConfigObject(canaryConfig);
       }
 
+      const totalGroupWeights: number = Object.values(this.stores.configEditorStore.computedGroupWeights).reduce(
+        add,
+        0
+      );
+      if (totalGroupWeights !== 100) {
+        log.error(`Configuration Error: Group weights need to equal 100.`);
+        this.stores.errorStore.push({
+          heading: `Configuration Error: Group weights need to equal 100.`,
```

Comment:
Not necessary to change, but it might be more clear if specified that the weights need to `add up` to 100